### DISCUSSION TITLE
Implementation question reguarding GlobalMaxPoolGrad

### DISCUSSION THREAD

**Discussion by georgen117** [microsoft/onnxruntime#6831] [Other Q&A]
I have been adding training for the dnnl endpoint. I have been working on Pooling operators but when it came to GlobalMaxPoolGrad I don't see any implementation for GlobalMaxPoolGrad in gradient_builder.cc.

Is GlobalMaxPoolGrad implemented and I am missing it?
If it is not implemented is there a plan to implement it in the future?

**Comment by SherlockNoMad** [comment]
GlobalMaxPoolGrad is not implemented yet. 
You can check the list of training-supported ops in gradient_builder.h 

So far, we haven't got any plan to implement a gradient for this op, but we would welcome a contribution for this gradient!
